ddp-workspace/projects/ddp-sdk/src/lib/services/address.service.ts

```diff
@@ -97,7 +97,7 @@ export class AddressService extends UserServiceAgent<Address> {
     }
 
     public deleteAddress(addressOrGuid: Address | string): Observable<any> {
-        const guid = addressOrGuid instanceof Address ? addressOrGuid.guid : <string>addressOrGuid;
+        const guid = addressOrGuid instanceof Address ? addressOrGuid.guid : addressOrGuid as string;
         return this.deleteObservable('/profile/address/' + guid, null, true);
     }
 }
```

Comment:
Probably it would be better to move `AddressService`  under  `ddp-workspace\projects\ddp-sdk\src\lib\services`